src/extensions/bigbang/bigbang.go

```diff
@@ -43,9 +43,15 @@ func Run(tmpPaths types.ComponentPaths, c types.ZarfComponent) (types.ZarfCompon
 	cfg := c.Extensions.BigBang
 	manifests := []types.ZarfManifest{}
 
+	err, validVersionResponse := isValidVersion(cfg.Version)
+
+	if err != nil {
+		return c, fmt.Errorf("invalid Big Bang version: %s, parsing issue %s", cfg.Version, err)
+	}
+
 	// Make sure the version is valid.
-	if !isValidVersion(cfg.Version) {
-		return c, fmt.Errorf("invalid Big Bang version: %s, must be at least 1.53.0", cfg.Version)
+	if !validVersionResponse {
+		return c, fmt.Errorf("invalid Big Bang version: %s, must be at least 1.54.0", cfg.Version)
```

Comment:
```suggestion
		return c, fmt.Errorf("invalid Big Bang version: %s, must be at least %s", cfg.Version, bbMinRequiredVersion)
```
Since you already have `bbMinRequiredVersion` might as well do this.